docs/source/glossary.md

```diff
@@ -17,4 +17,19 @@ Functional Programming
   This contrasts with functions or methods that depend on variables that are not explicitly passed as an input (such as accessing `self.variable` inside a method) or that alter the inputs or other state variables in-place, instead of returning new distinct variables as outputs.
 Dispatching
   Choosing which function or method implementation to use based on the type of the input variables (usually just the first variable). For some examples, see Python's documentation for the [singledispatch](https://docs.python.org/3/library/functools.html#functools.singledispatch) decorator.
-:::::
+
+[Equidispersion](http://www.ce.memphis.edu/7012/L20_CountDataModels_v2.pdf) (http://cursos.leg.ufpr.br/rmcd/introduction.html)
+  If in a Poisson distribution if the variance equals the mean of the distribution, it is reffered to as equidispersion.
+
+Generalized Poisson PMF
+  A generalization of the Poisson distribution, with two parameters X1, and X2, is obtained as a limiting form of the generalized negative binomial distribution. The variance of the distribution is greater than, equal to or smaller than the mean according. as X2 is positive, zero or negative. For formula and more detail visit the link (https://www.sciencedirect.com/science/article/pii/S0047259X14000256)
+
+[Bayes' theorem] (https://www.investopedia.com/terms/b/bayes-theorem.asp)
+  Describes the probability of an event, based on prior knowledge of conditions that might be related to the event. For example, if the risk of developing health problems is known to increase with age, Bayes' theorem allows the risk to an individual of a known age to be assessed more accurately (by conditioning it on their age) than simply assuming that the individual is typical of the population as a whole.
+
+Markov Chain (MC)
+  A Markov chain or Markov process is a stochastic model describing a sequence of possible events in which the probability of each event depends only on the state attained in the previous event. For a visual explantation, visit (https://setosa.io/ev/markov-chains/)
+
+[Markov Chain Monte Carlo (MCMC)] (https://machinelearningmastery.com/markov-chain-monte-carlo-for-probability/)
+  Markov chain Monte Carlo (MCMC) methods comprise a class of algorithms for sampling from a probability distribution. By constructing a Markov chain that has the desired distribution as its equilibrium distribution, one can obtain a sample of the desired distribution by recording states from the chain.  Various algorithms exist for constructing chains, including the Metropolis–Hastings algorithm. Various algorithms exist for constructing chains, including the Metropolis–Hastings algorithm.
```

Comment:
```suggestion
  Markov chain Monte Carlo (MCMC) methods comprise a class of algorithms for sampling from a probability distribution. By constructing a {term}`Markov chain` that has the desired distribution as its equilibrium distribution, one can obtain a sample of the desired distribution by recording states from the chain.  Various algorithms exist for constructing chains, including the Metropolis–Hastings algorithm. Various algorithms exist for constructing chains, including the Metropolis–Hastings algorithm.
```

Reply:
I am sorry but I am not still able to understand what you mean by link between the terms and I am not able to find the suggestion link. If you could add it in a comment in here, that'd be great.

Reply:
thanks for the help, could tell me if this is acceptable??